05week/spaceTravelToMars.js

```diff
@@ -11,6 +11,58 @@ let jobTypes = {
 
 // Your code here
 
+//you wnat CrewMember class
+// constructor should take in as input: name, their job, their specialSkill
+// have an attribute "ship", this is the ship they are in
+
+class CrewMember {
+  constructor(name, job, specialSkill){
+      this.name = name;
+      this.job = job;
+      this.specialSkill = specialSkill;
+      this.ship = null;
+  }
+
+//this method should ass THIS crewmember to the ship being passed in
+//NOTE: an entire ship instance is passed in, not just the name
+//NOTE: the entire crewmember iS ADDED 
+  enterShip(someShip){
+        this.ship = someShip;
+        someShip.crew.push(this);
+    }
+  }
+
+class Ship {
+  constructor(name, type, ability){
+    this.name = name;
+    this.type = type;
+    this.ability = ability;
+    this.crew = [];
+//have a list of crew that starts out empty "crew = [];"
+  }
+//this method should return the ship's ability if there is a crew 
+//whose job matches up with the ships type
+//otherwise should return "Cant perform a mission statement yet."
+  missionStatement(){
+    if(this.crew.length === 0){
+      return "Can't perform a mission yet."
+    }
+    else {
+      return this.ability;
```

Comment:
this method should check that there is a crewmember whose job matches the ships type